//!!!!!!!!!Репета
new Promise(((o,e)=>{const n=Math.random()>.5;setTimeout((()=>{n&&o("your order"),e("no finished your order")}),1e3)})).then((function(o){console.log("onMakeOrderSuccess"),console.log(o)})).catch((function(o){console.log("onMakeOrderError"),console.log(o)}));
//# sourceMappingURL=03-promises.7782b55b.js.map
